### PR TITLE
Sanaullah Language API

### PR DESCRIPTION
**Pull request by Sana-Ullah786**
This PR will be adding the endpoints of **Language Model**. This includes:

- Creating a New Language and Transacting it into the database in **language Table**. Using **POST METHOD**.
- Getting All Existing Languages in the database using **GET METHOD**.
- Getting a Specific Language from database using **GET METHOD WITH LANGUAGE ID**.
- Updating a Specific Language in database using **PUT METHOD WITH LANGUAGE ID**.
- Deleting a Specific Language from database using **DELETE METHOD WITH LANGUAGE ID**.
Please have a look at the **Endpoints** and **Test Cases**. Let me know about any changes. 👀 

**Ticket links:**
https://app.asana.com/0/1204411152293714/1204470953803097/f
https://app.asana.com/0/1204411152293714/1204470953803094/f
https://app.asana.com/0/1204411152293714/1204470953803100/f
https://app.asana.com/0/1204411152293714/1204470953803086/f

### CODE REVIEW

**Review by mustansirMuz — APPROVED** [review]
LGTM. Tests are failing because you haven't put the JWT env variables in circleci config. You can either wait for my PR to merge, so you can get that in your PR, or you can add them in yourselves and take care of the conflict while merging. Up to you.

**Comment by Sana-Ullah786** [comment]
Okay I'll.

**Review by mtalhatahir2001 — APPROVED** [review]
Just added a comment please review it before merging.

**Pull request merged**
